book/chapters/chapter10/advanced_technical_aspects_of_mlr3.qmd

```diff
@@ -551,6 +551,20 @@ aggr = benchmark(benchmark_grid(
   rsmp("cv", folds = 3)))$aggregate(conditions = TRUE)
 aggr[, .(learner_id, warnings, errors, classif.ce)]
 ```
+```{r technical-024-hidden}
+#| include: false
+lrn_debug = lrn("classif.debug", error_train = 0.3)
+lrn_debug$fallback = lrn("classif.featureless")
+
+withr::with_seed(
+  code = {aggr = benchmark(benchmark_grid(
+    tsk_penguins,
+    list(lrn_debug, lrn("classif.rpart")),
+    rsmp("cv", folds = 3)))$aggregate(conditions = TRUE)},
+  seed = 1
+)
+aggr[, .(learner_id, warnings, errors, classif.ce)]
+```
```

Comment:
We should *never* `set.seed` without telling the reader. Why was this needed here?